### ISSUE TITLE
False Positive detection on customElements

### ISSUE THREAD

**Issue by DDuran19** [astralhpi/svelte-fast-check#34]
Great work, Thanks for this package! 

Just got a problem with customElement, it reports a false positive (unless, I messed up somewhere in my code)

```bash
$ pnpm check
$ svelte-kit sync && svelte-fast-check --incremental && pnpm lint:biome
🔍 svelte-fast-check: Starting type check...

Converted 1 files, skipped 482 unchanged
📋 Diagnostics:

src/lib/client/googleMaps/GoogleMapsPin.svelte:2:17
Warning: The `customElement` option is used when generating a custom element. Did you forget the `customElement: true` compile option?
https://svelte.dev/e/options_missing_custom_element (svelte)
> <svelte:options customElement="gmap-pin" />
                  ^

────────────────────────────────────────────────────────────
Found 1 warning(s)
Completed in 5928ms
   (typeCheck: 696ms, svelteWarnings: 1150ms)
```

```svelte
<svelte:options customElement="gmap-pin" />

<script lang="ts">
	import Building from '@lucide/svelte/icons/building';
</script>

<Building class="bg-white" color="green" size="2rem" />
```

but I have that in my `svelte.config.js`

```js
import adapter from '@sveltejs/adapter-cloudflare';
import { vitePreprocess } from '@sveltejs/vite-plugin-svelte';

/** @type {import('@sveltejs/kit').Config} */
const config = {
	// ...rest of my configs,

	compilerOptions: {
		customElement: true
	}
};

export default config;

```

**Comment by webbrain-one** [comment]
Thanks for reporting this! I’ve opened a PR to fix the false positive warning you’re seeing with the `customElement` option. The update ensures that `svelte-fast-check` properly respects `compilerOptions.customElement: true` in your `svelte.config.js`, so you won’t see that warning anymore.

PR: https://github.com/astralhpi/svelte-fast-check/pull/35

Thanks again for the detailed report and for using the package! Let me know if you run into anything else. 🚀
